Common/src/main/java/at/petrak/hexcasting/api/casting/iota/Vec3Iota.java

```diff
@@ -29,7 +29,7 @@ public Vec3 getVec3() {
     @Override
     public boolean isTruthy() {
         var v = this.getVec3();
-        return v.x != 0.0 && v.y != 0.0 && v.z != 0.0;
+        return v.x != 0.0 || v.y != 0.0 || v.z != 0.0;
```

Comment:
```suggestion
        return !(v.x == 0.0 && v.y == 0.0 && v.z == 0.0);
```
I feel like this conveys the intent a bit better, right? ie. only the zero vector is falsy, everything else is truthy. (assuming i remember how Java bool syntax works)